go/vt/vttablet/grpctmserver/server_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package grpctmserver
+package grpctmserver_test
```

Comment:
made this change so i can call RegisterForTest and not generate an import cycle between here and the grpctmclient tests